doc/website/advanced/custom-iceoryx-platform.md

```diff
@@ -0,0 +1,37 @@
+# Custom iceoryx platforms
+
+The `iceoryx_hoofs/platform` represents the lowest layer in iceoryx. Its task is to establish
+a uniform behavior of low level system calls across all supported platforms. When one would like
+to add support for another platform one can use the CMake argument `-DPLATFORM_PATH` to provide
+a path to a custom platform implementation.
+This may become necessary to support an unsupported compiler for a specific
+platform, adjust system calls to become posix compliant or to add a new operating system.
+
+## Build with custom iceoryx platform
+
+Let's assume you have your platform stored in the absolute path `/home/user/newIceoryxPlatform`
+you can configure cmake with the command:
+```sh
+cd iceoryx
+cmake -Bbuild -Hiceoryx_meta -DPLATFORM_PATH=/home/user/newIceoryxPlatform
```

Comment:
Can you make this `IOX_PLATFORM_PATH`/`ICEORYX_PLATFORM_PATH` please?

Reply:
... you can of course do this in the last PR if it's easier

Reply:
yes no problem